gto-support-db-livedata/src/main/kotlin/org/ccci/gto/android/common/db/LiveDataDao.kt

```diff
@@ -70,17 +70,7 @@ class LiveDataRegistry {
 
     @MainThread
     internal fun DaoComputableLiveData<*>.registerFor(query: Query<*>) {
-        registerFor(query.table)
-        query.joins.forEach { registerFor(it) }
-    }
-
-    @MainThread
-    private fun DaoComputableLiveData<*>.registerFor(table: Table<*>) = registerFor(table.type)
-
-    @MainThread
-    private fun DaoComputableLiveData<*>.registerFor(join: Join<*, *>) {
-        join.base?.let { registerFor(it) }
-        registerFor(join.target)
+        query.allTables.forEach { registerFor(it.type) }
```

Comment:
Are these tables types the type that is referenced in the invalidationFlow? So it will be a collection of table types?

Reply:
yes, from a `Query` object we look up all Tables it references. Each table corresponds to an object type.